dev/ci/docker/edge_ubuntu/Dockerfile

```diff
@@ -15,16 +15,15 @@ RUN apt-get update -qq && apt-get install --no-install-recommends -y -qq \
         perl libgmp-dev libgmp-dev:i386 \
         # Dependencies of lablgtk (for CoqIDE)
         libgtksourceview-3.0-dev adwaita-icon-theme-full \
-        # Dependencies of Gappa
-        libboost1.81-all-dev libmpfr-dev autoconf-archive bison flex \
         # Dependencies of source-doc and coq-makefile
         texlive-latex-extra texlive-science tipa \
         # Dependencies of HB (test suite)
         wdiff \
         # Required to get the wget step to succeed
         ca-certificates \
         # Required for fiat-crypto and Coqtail
-        python-is-python3
+        python-is-python3 \
+      && rm -rf /var/lib/apt /usr/share/doc
```

Comment:
Removing /var/lib/apt may prove too heavy handed, if for example one want to log into the image to debug apt installs; for 60MiB I would keep it, but do as you wish.

Note that several subdirs in `var/lib/apt` can be indeed removed without problem (such as `/var/lib/apt/lists`), which is usually the culprit for size problem there.